bindings/python/iota_sdk/types/block.py

```diff
@@ -17,20 +18,24 @@ class Block:
 
     protocolVersion: int
     parents: List[HexStr]
-    nonce: int
+    nonce: str
     payload: Optional[Payload] = None
 
     @classmethod
     def from_dict(cls, block_dict: Dict) -> Block:
-        obj = cls.__new__(cls)
-        super(Block, obj).__init__()
-        for k, v in block_dict.items():
-            setattr(obj, k, v)
-        return obj
+        return from_dict(Block, block_dict)
```

Comment:
I don't know Python but that kinda feels a bit recursive

Reply:
It's probably not but just checking

Reply:
It's not :+1:

Reply:
Thanks